app/core/services/opportunities/opportunities.service.js

```diff
@@ -4,17 +4,19 @@ angular
     "$resource",
     "$q",
     "JwtService",
+    'API_ENDPOINTS',
     class {
       #opportunities;
       #SIGNED_IN_USER_ID;
 
-      constructor($resource, $q, JwtService) {
+      constructor($resource, $q, JwtService, API_ENDPOINTS) {
         this.$q = $q;
         this.#SIGNED_IN_USER_ID = JwtService.getToken().value;
         this.$resource = $resource;
+        this.API_ENDPOINTS = API_ENDPOINTS
 
         this.#opportunities = $resource(
-          "http://localhost:3000/opportunities?_sort=createdAt&_order=desc"
+          API_ENDPOINTS.OPPORTUNITIES + "?_sort=createdAt&_order=desc"
```

Comment:
it would be beneficial to have a namespace telling what `?_sort=createdAt&_order=desc` means. Even tough nowadays it is obvious to us what is its meaning, it may be a hard task to figure out its meaning in the future.